An initial design

mukilane opened the pull request:
Hey folks, 
I have

imkarthikk commented:
Absolutely brilliant. Do you think you can add some more description for each and every commit. Will help us better. :)

swaathi commented on _config.yml:
```diff
@@ -0,0 +1,21 @@
+baseurl: "" # the subpath of your site, e.g. /blog
+url: "http://gdgchennai.com" # the base hostname & protocol for your site
+twitter_username: GTUGChennai                   
+github_username:  skript
```
The Github username is `skcript`. And the Twitter handle is `GDGChennai`. Can you please change that?

mukilane replied:
Sure.

swaathi commented on _includes/footer.html:
```diff
@@ -0,0 +1,20 @@
+    </ul>
+    </div>
+    <div class="mdl-mini-footer__right-section">
+      <a class="mdl-mini-footer__social-btn"  href="https://github.com/skript/gdgchennai"><img src=""></a>
```
The Github link is, `https://github.com/skript/gdgchennai`. Please change.

imkarthikk commented on _posts/2016-06-18-welcome-to-jekyll.markdown:
```diff
@@ -0,0 +1,25 @@
+---
```
I think this file can be removed. We don't use in-house posts for sharing blogposts. We use medium instead.

mukilane replied:
@imkarthikk , I just kept that to test the post layout. I'll remove it. Lots of files are yet to be cleaned. I'll check on that.

And I have an idea of moving the http://gtugchennai.blogspot.in/ blog to jekyll. 
It looks abandoned there, and the process of integrating is not so difficult.

imkarthikk replied:
Wonderful job. We're going to be moving that to medium: https://medium.com/gdgchennai/. If you could help us move there, that would be wonderful. 

BTW, what you are doing with your time is stunning. Keep up this enthusiasm. :)

mukilane replied:
Contributing makes me learn new things too.

swaathi commented:
@mukilane Can you resolve this?
@imkarthikk and @varun-raj Any inputs?

mukilane commented:
@swaathi 
I have resolved it.

imkarthikk commented:
@mukilane : I need two things. 

1. Can you post a screenshot of the design you've done? That way, everyone gets to see without having to clone and run your pull. 
2. Can you shoot me an email to karthik at skcript.com ?

mukilane commented:
@imkarthikk 
1. I'll post the SS shortly.
2. Done.

mukilane commented:
Build 27/6

![SS 1](https://cloud.githubusercontent.com/assets/5374364/16416016/c87142b0-3d5d-11e6-9d33-bec0a93e7b01.png)
![SS 2](https://cloud.githubusercontent.com/assets/5374364/16416103/3d181300-3d5e-11e6-80e8-4955ebdc6d93.png)
![SS 3](https://cloud.githubusercontent.com/assets/5374364/16416122/5f9fc724-3d5e-11e6-9293-b3ea86ca06d3.png)

mukilane commented:
About page 
![About page](https://cloud.githubusercontent.com/assets/5374364/16416710/313707b4-3d61-11e6-81a9-f1b522b2e015.png)

Share dialog
![Share dialog](https://cloud.githubusercontent.com/assets/5374364/16416751/575ec49a-3d61-11e6-87c5-4e774d203f21.png)

mukilane commented:
**Change log** - [Build 3/7](https://github.com/skcript/gdgchennai/pull/7/commits/16fc3f14b56ee9329dd2c81e3bd8fe82774d0c57) 

1. Added Contacts page (typeform embed)
2. Added a Posts page which displays links to Medium posts
3. Added a _data folder which contains ymls for posts, links and members. 
4. Added favicon. 
5. Updated the slider, page layout, tooltip.

imkarthikk commented:
@mukilane : Alright. I'm going to go ahead and run your fork locally and see how we can improve it and then push it live. :)

mukilane commented:
@imkarthikk . 
Just to make the testing process easier, I have just deployed the site to Firebase. 
https://gdgchennai-14b7c.firebaseapp.com

**Updates:** Build 5/9
1. I have included some of the features of PWA.  Added Web Manifest 
2. So if on HTTPS, the site will prompt to "Add to Shelf" (on Desktop) or "Add to Homescreen" (on Mobile)
3. Adder Service Workers. Enabled caching for common site assets (js, css)
4. Other small corrections ( a lighter logo for the footer, styling issues, removed unwanted sass, etc )

imkarthikk commented:
@mukilane : 
Fantastic. I'll test it out and report some bugs. :)

mukilane commented:
**Changelog**: Build 2/10
1. Added AJAX navigation between pages .
2. Added a new Events page
3. Optimized most of the images to decrease load time.
4. Slightly tweaked the home page with some stats.
